lib/apiservers/engine/backends/container.go

```diff
@@ -1246,8 +1249,11 @@ func (c *Container) ContainerStop(name string, seconds *int) error {
 	operation := func() error {
 		return c.containerProxy.Stop(vc, name, seconds, true)
 	}
-	if err := retry.Do(operation, IsConflictError); err != nil {
-		return err
+
+	config := retry.NewBackoffConfig()
+	config.MaxElapsedTime = maxElapsedTime
+	if err := retry.DoWithConfig(operation, IsConflictError, config); err != nil {
+		return ConflictError(err.Error() + " --- Retrying the operation may resolve the issue.")
```

Comment:
This may not be a `ConflictError`, so better just to leave it as it it was before.

Reply:
Do you mean `return err`?

Reply:
Yes.